REVIEW_GUIDELINE.md

```diff
@@ -0,0 +1,255 @@
+# Guideline for Pull Request Review
+
+## Introduction
+
+Code review is an essential part of the software development life cycle and is also one of the most challenging and time-consuming aspects of the software development process, requiring experienced team members to spend time reading, thinking about, assessing, and responding to new feature, fix, and system implementations.
+
+Here are some guidelines to assist you in reviewing pull requests.
+
+## Guidelines
+
+Before jumping straight to the code changes, some obvious things should be addressed first.
+
+**Pull Request Checklist:**
+
+1. Commit Message (spell checks, commit lint etc.)
```

Comment:
Please add a link for the commit message guideline for reference https://github.com/innovaccer/design-system/blob/master/CONTRIBUTING.md#commit-guidelines